ygen/testdata/proto/proto-test-e.uncompressed.enums.formatted-txt

```diff
@@ -0,0 +1,34 @@
+// openconfig.enums is generated by codegen-tests as a protobuf
+// representation of a YANG schema.
+//
+// Input schema modules:
+//  - testdata/proto/proto-test-e.yang
+// Include paths:
+syntax = "proto3";
+
+package openconfig.enums;
+
+import "github.com/openconfig/ygot/proto/ywrapper/ywrapper.proto";
+import "github.com/openconfig/ygot/proto/yext/yext.proto";
+
+// ProtoTestE_Animals_Animal_Config_Species represents an enumerated type generated for the YANG typedef .
```

Comment:
stray space before '.' Here and elsewhere.

Reply:
Fixed, this was an issue with how the name of synthesised entries' types were populated, fixed.